jsonify.go

```diff
@@ -44,40 +37,28 @@ func (t *Table[V]) MarshalJSON() ([]byte, error) {
 }
 
 // DumpList dumps ipv4 od ipv6 tree into list of roots and their children
```

Comment:
still a typo in this public docstring and a missing dot at end of line. If this method has to be public, please invest more time to find a clear description or spend an extra test or maybe a dumplist_example_test.

Reply:
I've fixed both docstrings.

I don't think that we need separated tests for DumpList. It will be duplicate of json tests.

Reply:
The test for DumpList was not meant for testing, just to better explain the public method, but I can live without or add it myself later.

Two more small remarks:
- public type ListElement needs a docstring.
- please rename field Cidr to CIDR, for golang naming convention, like HTTP and not Http
Regards
   Charly

Reply:
Done